workspace.json

```diff
@@ -2,6 +2,7 @@
   "$schema": "./node_modules/nx/schemas/workspace-schema.json",
   "version": 2,
   "projects": {
+    "cypress-mock-openapi": "packages/cypress-mock-openapi",
```

Comment:
Were there any differences in the build output of the original package and this one?

Reply:
No, as the build is only copying files, I replicated the same build result.